_layouts/product-overview.html.slim

```diff
@@ -27,10 +27,15 @@ br
   div(class="feature-text")
     = page._features3
 
+= page._new_feature
```

Comment:
You don't need to do this. Just add the content to `overview.adoc` as a new level-3 (===) section at the end of the `detail` section.